.github/workflows/nuxt-nightly.yml

```
@@ -0,0 +1,29 @@
+name: nuxt-nightly
+
+on:
+  # schedule:
+  #   - cron: "0 5 * * 1,3,5" # monday,wednesday,friday 5AM
+  workflow_dispatch:
+  push:
+    branches:
+      ci/nightly
+
+jobs:
+  test:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: corepack enable
+      - uses: actions/setup-node@v4
```

```suggestion
      - uses: actions/setup-node@v3
```

because bugs v4

https://github.com/actions/setup-node/issues/878